security/fma-kona-asterisc.md

```diff
@@ -0,0 +1,213 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Stage 1.4: Kona and Asterisc FMA (Failure Modes and Recovery Path Analysis)](#stage-14-kona-and-asterisc-fma-failure-modes-and-recovery-path-analysis)
+  - [Introduction](#introduction)
+  - [Failure Modes and Recovery Paths](#failure-modes-and-recovery-paths)
+    - [Kona and Asterisc Hot Path Failure](#kona-and-asterisc-hot-path-failure)
+    - [Divergent Kona Derivation Pipeline](#divergent-kona-derivation-pipeline)
+    - [Kona Execution Diverges from op-program](#kona-execution-diverges-from-op-program)
+    - [Breaking Backwards Compatibility for the op-challenger](#breaking-backwards-compatibility-for-the-op-challenger)
+    - [Divergence between RISCV.sol and Asterisc](#divergence-between-riscvsol-and-asterisc)
+    - [Generic items we need to take into account:](#generic-items-we-need-to-take-into-account)
+  - [Audit Requirements](#audit-requirements)
+  - [Appendix](#appendix)
+    - [Appendix A: op-challenger](#appendix-a-op-challenger)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Stage 1.4: Kona and Asterisc FMA (Failure Modes and Recovery Path Analysis)
+
+| | |
+|--------|--------------|
+| Author | Andreas Bigger |
+| Created at | 2024-10-17 |
+| Initial Reviewers | Matt Solomon, Ethnical |
+| Need Approval From | Ben Clabby, Mark Tyneway |
+| Status | In Review |
+
+
+## Introduction
+
+This document covers the initial deployment of [the Asterisc Fault Proof VM][asterisc] with [the Kona Fault Proof Program][kona].
+
+Together, Kona and Asterisc form an alternative proof stack to the [op-program] and [cannon].
+A secondary dispute game will be created that uses kona and asterisc.
```

Comment:
>A secondary dispute game will be created that uses kona and asterisc.

Does this means that each time we call `create()` on the DGF this will also create a second game? 
Looks like gas heavy on L1 to me. Also,  hard to code something that manage both games in the same times.

Reply:
No. This release does not enable multi-proofs, kona/asterisc will exist as a fully separate dispute game, like the `PermissionedDisputeGame`.